cmd/action/migrate.go

```diff
@@ -0,0 +1,179 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"text/template"
+	"time"
+
+	"ariga.io/atlas/sql/migrate"
+	"ariga.io/atlas/sql/schema"
+	"github.com/spf13/cobra"
+)
+
+const (
+	migrateFlagDir        = "dir"
+	migrateFlagSchema     = "schema"
+	migrateFlagForce      = "force"
+	migrateDiffFlagDevURL = "dev-url"
+	migrateDiffFlagTo     = "to"
+)
+
+var (
+	// MigrateFlags are the flags used in MigrateCmd (and sub-commands).
+	MigrateFlags struct {
+		DirURL  string
+		DevURL  string
+		ToURL   string
+		Schemas []string
+		Force   bool
+	}
+	// MigrateCmd represents the migrate command. It wraps several other sub-commands.
+	MigrateCmd = &cobra.Command{
+		Use:   "migrate",
+		Short: "Manage versioned migration files",
+		Long:  "`atlas migrate`" + ` wraps several sub-commands for migration management.`,
```

Comment:
```suggestion
		Long:  "'atlas migrate' wraps several sub-commands for migration management.`,
```

I changed it in apply, and I want to change in other places as well.